post/getting-started-preact/index.md

```diff
@@ -0,0 +1,322 @@
+---
+title: "Getting Started with Directus and Preact"
+description: "120-160 characters"
+author:
+    name: "jay bharadia"
+    avatar_file_name: "my-avatar.jpg"
+---
+
+## Introduction
+
+Fast 3kB alternative to React with the same modern API. In this tutorial, you will learn how to build a website using Directus as a headless CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
+
+Before You Start
+
+You will need:
+
+To install Node.js and a code editor on your computer.
+
+A Directus project - you can use Directus Cloud or run it yourself.
+
+Some knowledge of Preact framework
+
+## Initializing Preact
+
+The modern way to get started is using Vite bundler instead of old preact-cli
+
+## Using Global Metadata
+
+In your Directus project, navigate to Settings -> Data Model and create a new collection called global. Under the Singleton option, select 'Treat as a single object', as this collection will have just a single entry containing global website metadata.
+
+Create two text input fields - one with the key of title and one description.
+
+Navigate to the content module and enter the global collection. Collections will generally display a list of items, but as a singleton, it will launch directly into the one-item form. Enter information in the title and description field and hit save.
+
+By default, new collections are not accessible to the public. Navigate to Settings -> Access Control -> Public and give Read access to the Global collection.
+
+![Global Metadata](./images/global-metadata.png)
+
+## Setup Routing
+
+We will load Blog Post List, detail page, dynamic pages and a 404 page.
+
+```js
+import { render } from "preact";
+import { LocationProvider, Router, Route } from "preact-iso";
+
+import { NotFound } from "./pages/_404.jsx";
+import { BlogList } from "./pages/blog/list.jsx";
+import { Blog } from "./pages/blog/slug.jsx";
+import { Page } from "./pages/slug.jsx";
+import { Home } from "./pages/index.jsx";
+
+import { Header } from "./components/Header.jsx";
+import "./style.css";
+export function App() {
+    return (
+        <LocationProvider>
+            <Header />
+            <main>
+                <Router>
+                    <Route path="/" component={Home} />
+
+                    <Route path="/blog" component={BlogList} />
+                    <Route path="/404" component={NotFound} />
+                    <Route path="/blog/:slug" component={Blog} />
+                    <Route path="/:slug" component={Page} />
+                </Router>
+            </main>
+        </LocationProvider>
+    );
+}
+render(<App />, document.getElementById("app"));
+```
```

Comment:
Please follow the standard format exactly - load global metadata in an otherwise-empty page please.